src/palace/manager/service/container.py

```diff
@@ -91,8 +91,8 @@ def wire_container(container: Services) -> None:
             "palace.manager.api.overdrive",
             "palace.manager.feed.annotator.circulation",
             "palace.manager.feed.acquisition",
+            "palace.manager.metadata_layer.policy.replacement",
```

Comment:
Had to update the path in the container where we are doing a dependency injection.